redfish-core/lib/FabricAdapters.hpp

```diff
@@ -269,7 +269,7 @@ inline void getAdapter(const std::shared_ptr<bmcweb::AsyncResp>& aResp,
                 if (ec.value() == boost::system::errc::io_error)
                 {
                     messages::resourceNotFound(
-                        aResp->res, "#FabricAdapter.v1_2_0.FabricAdapter",
+                        aResp->res, "#FabricAdapter.v1_4_0.FabricAdapter",
```

Comment:
upstream fxied this, these should not have the version in them, oh well. 
https://github.com/openbmc/bmcweb/search?q=resourceNotFound